### PR TITLE
New filter icon and align collapsible rows

### PR DESCRIPTION
**Pull request by sroy3**
screen shot 2016-06-29 at 4 08 46 pm
* Changed the filter box clear icon
* Aligned collapsible rows in mod-align-header
* Changed dropdown active color

### CODE REVIEW

File: scss/controls/filter-box.scss [hunk 1]
```diff
@@ -22,8 +22,8 @@
 
     & + span {
       position: absolute;
-      top: 8px;
-      right: 20px;
+      top: 12px;
```
Comment (JPLachance) [inline scss/controls/filter-box.scss]: Couldn't we simply do a `calc(50% - #{$x-icon-height} / 2)`. It would be better in my opinion.

**Comment by JPLachance** [comment]
🚢 :it:!

File: scss/controls/filter-box.scss (hunk 1 above)
Comment (sroy3) [inline scss/controls/filter-box.scss]: Done

**Comment by wfortin** [comment]
Nice!

**Pull request merged**